MekHQ/src/mekhq/gui/model/PersonnelTableModel.java

```diff
@@ -66,7 +66,7 @@
     private boolean loadAssignmentFromMarket;
     private boolean groupByUnit;
 
-    private final MekHqColors colors = new MekHqColors();
+    private static final transient MekHqColors colors = new MekHqColors();
```

Comment:
?

Reply:
Was testing something out, flipped back.
```suggestion
    private final MekHqColors colors = new MekHqColors();
```